src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -515,6 +515,8 @@ public String toString() {
     sb.append(gameName);
     sb.append(", gameVersion=");
     sb.append(gameVersion);
+    sb.append(", loader=");
+    sb.append(loader);
```

Comment:
Point taken.  However, prior to the introduction of the serialization proxy feature a few weeks ago, `GameData` didn't override `toString()`, and thus it always returned a string of the form `games.strategy.engine.data.GameData@<hashCode>`.  It's unlikely any code would have been specifically using that encoded form.  I only added the override in order to have useful information in test failure messages when attempting to round-trip `GameData` objects through the new proxy serialization framework.

There are currently 20 calls to `Object#toString()` in the code.  Most of them can be eliminated from ever possibly calling `GameData#toString()` based on the surrounding context (e.g. calling `toString()` on the values in a `JComboBox<Object>` in which it's obvious the objects are players).  The one call where I couldn't totally rule it out in `PropertyUtil`, the result of the `toString()` call is used simply to format an error message.

So I'll leave you with I'm 95% sure `GameData#toString()` is not used for anything important. :smile: